### PR TITLE
Initiative additional resources and linking actiontype

### PR DESCRIPTION
**Pull request by shavnita**

### CODE REVIEW

File: wp-content/themes/oceanwp-child/assets/css/template-styles.css [hunk 1]
```diff
@@ -211,6 +211,14 @@ hr{
...
 }
 
+.news-tag{
+    border: 1px solid #ccc;
+    border-radius: 8px;
+    padding:8px;
+    margin-right:5px;
+    background-color:#5ba3bb94;
+    color:#275f87
+}
```
Comment (YonatanKra) [inline wp-content/themes/oceanwp-child/assets/css/template-styles.css:221]: This should be dropped.

File: wp-content/themes/oceanwp-child/functions.php [hunk 2]
```diff
@@ -217,15 +296,27 @@ function custom_repeat_meta_box_save($post_id) {
 		$new = array();
 		$title = $_POST['UpdateTitle'];
         $date = $_POST['UpdateDate'];
-		$updates = $_POST['Update'];
-		 $count = count( $updates );
-		 for ( $i = 0; $i < $count; $i++ ) {
+		$updates = $_POST['Update'];		
+		$actiontype = $_POST['ActionType'];
+		$links = $_POST['ActionLink'];
```
Comment (YonatanKra) [inline wp-content/themes/oceanwp-child/functions.php:301]: That's a bit strange to me. Can you explain how this POST data is coming? Aren't we using Ajax or getting data from wordpress DB?

File: wp-content/themes/oceanwp-child/single-initiative.php [hunk 3]
```diff
@@ -191,22 +191,22 @@
 								<div class="col-md-8 col-lg-8 col-sm-12">
 									<div class="row">
 						<?php 
-							$the_post_id = get_the_ID();
-							$actions = wp_get_post_terms($the_post_id, 'actiontype', ['']);
-							
-							if(empty($actions) || ! is_array($actions)){
-								echo "No Actions";
+							$initiative_actiontypes = get_post_meta($the_post_id, 'initiative');
```
Comment (YonatanKra) [inline wp-content/themes/oceanwp-child/single-initiative.php:194]: Change variable to `$initiative_actions_types`

File: wp-content/themes/oceanwp-child/comments.php [hunk 4]
```diff
@@ -0,0 +1,141 @@
+<?php
```
Comment (YonatanKra) [inline wp-content/themes/oceanwp-child/comments.php:1]: Remove file

**Review by YonatanKra — CHANGES_REQUESTED** [review]
Note the inline comments

**Comment by YonatanKra** [comment]
image
Notice the icon near the country name. Should it be like this?

File: wp-content/themes/oceanwp-child/functions.php (hunk 2 above)
Comment (shavnita) [reply wp-content/themes/oceanwp-child/functions.php:301]: These are coming from custom meta boxes created for initiatives updates and action type. I had mentioned earlier that I have created updates function from the news meta boxes function you had created.

File: wp-content/themes/oceanwp-child/single-initiative.php (hunk 3 above)
Comment (shavnita) [reply wp-content/themes/oceanwp-child/single-initiative.php:194]: Done

**Review by YonatanKra — DISMISSED** [review]

**Review by YonatanKra — APPROVED** [review]
Seems OK.
Please pull it and make sure it is working correctly.

**Pull request merged**